internal/rule/rule.go

```diff
@@ -132,6 +132,8 @@ func (rule *Rule) checkOutputFields(event *event.Event) bool {
 	return true
 }
 
+// TODO: remove? it is not used
```

Comment:
It will be, I prepared this function, but I need to add `tags` management in falcosidekick before, to allow to work with